Description: Avoid using rpath to find X11 libraries at run-time.
Author: Anthony Fok
Forwarded: not-needed
Last-Update: 2018-06-30

--- a/ft2demos/graph/x11/rules.mk
+++ b/ft2demos/graph/x11/rules.mk
@@ -89,9 +89,11 @@
   # The GRAPH_LINK variable is expanded each time an executable is linked
   # against the graphics library.
   #
-  ifeq ($(PLATFORM),unix)
-    GRAPH_LINK += $(X11_LIB:%=-R%)
-  endif
+  # No, we do not want rpath on Debian; reversing YAMANO-UCHI Hidetoshi's
+  # 2003-06-13 change.  (Anthony Fok, 2003-08-28)
+  #ifeq ($(PLATFORM),unix)
+  #  GRAPH_LINK += $(X11_LIB:%=-R%)
+  #endif
   GRAPH_LINK += $(X11_LIB:%=-L%) -lX11
 
   # Solaris needs a -lsocket in GRAPH_LINK.
